tasks/src/audio-classification/data.ts

```diff
@@ -39,25 +39,34 @@ const taskData: TaskData = {
 	libraries: TASKS_MODEL_LIBRARIES["audio-classification"],
 	metrics:   [
 		{
-			description: "",
-			id:          "accuracy",
+			description: "The Accuracy metric is the ratio of correct predictions to the total number of cases processed. It can be calculated as: Accuracy = (TP + TN) / (TP + TN + FP + FN). Where TP is True Positive; TN is True Negative; FP is False Positive; and FN is False Negative.",
```

Comment:
As mentioned by @merveenoyan before, the metrics descriptions are unnecessary. If the metric is defined in hf.co/metrics and the `description` is empty, we pull the definition automatically from hf.co/metrics.

Reply:
Perfect, will apply.